src/components/MovieList.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import MovieListItem from './MovieListItem';
+
+export default class MovieList extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.isEmpty = !(this.props.movieList && this.props.movieList.length);
```

Comment:
Use state to store local component data which affects view.

```
constructor(props) {
        super(props);
        this.isEmpty = !(this.props.movieList && this.props.movieList.length);
        this.state = {
    isEmpty: !(this.props.movieList && this.props.movieList.length),
  }
}
```

Reply:
Done